features/steps/invitation_steps.js

```diff
@@ -15,6 +18,26 @@ When(
   }
 );
 
+When(
+  "{a} {invitation} for {a} {space} is accepted by {a} {actor}",
+  /**
+   * @param {Invitation} invitation
+   * @param {Space} space
+   * @param {Actor} actor
+   */
+  function (_, invitation, _2, space, _3, actor) {
+    return (
+      invitation
+        .rsvpLink()
+        // @todo Refactor to live in the harness
```

Comment:
I am not sure what is our workflow with TODOs like this one. Do we just leave them in the code for later? Do we make issues to remind ourselves to revisit this? Mostly I'm wondering how will we make sure that a TODO like this will eventually be either addressed (and removed), or declared to be not-worth-doing (and removed).

Reply:
I think we can tackle this before merging, since it's pretty concrete; and doesn't require adding new steps or implementing parts of the feature.